src/types.d.ts

```diff
@@ -105,4 +105,18 @@ type BundleMDXOptions = {
    * ```
    */
   globals?: Record<string, string | ModuleInfo>
+  /**
+   * The current working directory for the mdx bundle.
+   *
+   * All relative paths are resolved from this directory by esbuild. The supplied
+   * folder doesn't have to exist.
```

Comment:
Could you further describe in this comment when one would provide a folder that doesn't exist? What's the use case for that?

Reply:
Yep. Added some more info.